simple.go

```diff
@@ -18,7 +18,11 @@ func newSimpleCache(cb *CacheBuilder) *SimpleCache {
 }
 
 func (c *SimpleCache) init() {
-	c.items = make(map[interface{}]*simpleItem, c.size)
+	if c.size == -1 {
```

Comment:
Hi,
When do you guys expect this to be merged?
This is exactly the change I've been waiting for...

Reply:
@kogan69 Hi, my bad sort of forgot about it. I will complete the PR by Wednesday this week.